src/client/components/pages/index.js

```diff
@@ -212,20 +212,17 @@ class IndexPage extends React.Component {
 						Register!
 					</Button>
 				</div>
-				<RevisionsTable
-					results={this.props.recent}
-				/>
 				<div>
-					<Row>
-						<Col mdOffset={10}>
-							<Button
-								bsStyle="primary"
-								href="/revisions"
-							>
-								See all revisions
-							</Button>
-						</Col>
-					</Row>
+					<RevisionsTable
+						results={this.props.recent}
+					/>
+					<Button
+						bsStyle="primary"
+						className="pull-right"
```

Comment:
I think the button would be better if it was centered.
On mobile it works OK on the right, but on desktop size I feel it's not visible enough and I find myself looking for it.

You can wrap a `<div className="text-center">` around the button to center it.
What do you think?

Reply:
Like this?

![Screenshot from 2020-02-12 08-02-24](https://user-images.githubusercontent.com/45737735/74297837-0858cc80-4d6e-11ea-9222-b34860cb15dd.png)

Reply:
I think that works better yes. My eye naturally finds it at the end of the table

Reply:
> I think that works better yes. My eye naturally finds it at the end of the table

I'll make the change